chapters/sampling-randomization-power.tex

```diff
@@ -398,8 +450,77 @@ \subsection{Minimum sample size}
 than an output for reporting requirements.
 At the end of the day, you will probably have reduced
 the complexity of your experiment significantly.
-For reporting purposes, such as grantwriting and registered reports,
+For reporting purposes, such as grant writing and registered reports,
 simulation ensures you will have understood the key questions well enough
 to report standard measures of power once your design is decided.
 
-%------------------------------------------------
+\subsection{Randomization inference}
+
+Randomization inference is used to analyze the likelihood
+that the randomization process, by chance,
+would have created a false treatment effect as large as the one you observed.
+Randomization inference is a generalization of placebo tests,
+because it considers what the estimated results would have been
+from a randomization that did not in fact happen in reality.
```

Comment:
since we are very careful above to distinguish between randomized treatment assignment and random sample, best to continue to be precise with language here as well (e.g. "randomized treatment assignment that did not in fact happen..."

Reply:
this can still be either I think, hence nonspecific?